pkgs/by-name/ht/httptap/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  fetchpatch,
+  libpcap,
+}:
+
+buildGoModule rec {
+  pname = "httptap";
+  version = "0.0.8";
+
+  src = fetchFromGitHub {
+    owner = "monasticacademy";
+    repo = "httptap";
+    tag = "v${version}";
+    hash = "sha256-1BtV5ao5dAKSINdUdJD/wxTMFXXiP8Vy1A7gQfVIsUQ=";
+  };
+
+  patches = [
+    # this patch updates go.mod to include missing dependencies
+    # https://github.com/monasticacademy/httptap/pull/13
+    (fetchpatch {
+      name = "update-go-mod";
+      url = "https://github.com/monasticacademy/httptap/commit/3b520725c784d6435be6a51c58ae847bae729962.patch";
+      hash = "sha256-0dPq0Ldu1m8YZKctFtoUcbQdmx6sqjA8EVhTeMwNWx8=";
+    })
+  ];
+
+  vendorHash = "sha256-+TtHw2KdeNHCgnMnkxJJ9shqsrlbeTzYwbPH0dJmCjM=";
+
+  buildInputs = [
+    libpcap
```

Comment:
I am not sure we actually need this. The program compiles fine without it and doesn't spit any warnings.

Reply:
fair - will remove!

Reply:
updated!

Reply:
Even when turning cgo back on, it does not get linked.